pkg/infinispan/grafana/dashboard_controller.go

```diff
@@ -0,0 +1,103 @@
+package grafana
+
+import (
+	"context"
+
+	grafanav1alpha1 "github.com/integr8ly/grafana-operator/v3/pkg/apis/integreatly/v1alpha1"
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/discovery"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var log = logf.Log.WithName("infinispan-grafana")
+
+func AddToScheme(scheme *runtime.Scheme) error {
+	return grafanav1alpha1.AddToScheme(scheme)
+}
+
+func EnsureDashboardExported(context context.Context, namespace string, mgr manager.Manager) error {
```

Comment:
`CreateDashboard`?

Reply:
Or even just `Create`. That way you have `grafana.Create` when calling using the package name.

Reply:
Hrmm, okay